src/main/java/frc/robot/Constants.java

```diff
@@ -103,14 +102,17 @@ public static final class Climber {
         public static final double MAX_SETPOINT = 120;
         public static final double ZEROING_SPEED = -0.15;
         public static final double RIGHT_CURRENT_MAX = 7;
-        public static final double LEFT_CURRENT_MAX = 12;
+        public static final double LEFT_CURRENT_MAX = 17;
         public static final double TRIGGER_DEADBAND = 0.1; // TODO: set
         public static final double DOWN = 13;
     }
 
     public static final class AmpBar {
-        public static final double IN = 0.45;
-        public static final double OUT = IN + 0.2506;
+        public static final double IN = 0;
+        public static final double OUT = -0.625;
+        public static final double WHEEL_SPEED = -0.5;
+        public static final double ERROR_OF_MARGIN = 0.1;
```

Comment:
This margin of error is 40% of the encoder distance between IN and OUT. You might want to consider tightening this margin.